indexing-service/src/test/java/org/apache/druid/indexing/common/actions/UpdateStatusActionTest.java

```diff
@@ -78,12 +81,20 @@ public void testTaskStatusFull()
   @Test
   public void testNoTaskRunner()
   {
-    UpdateStatusAction action = new UpdateStatusAction("successful");
+    UpdateStatusAction action = new UpdateStatusAction("", TaskStatus.success(ID));
     Task task = NoopTask.create();
     TaskActionToolbox toolbox = mock(TaskActionToolbox.class);
     TaskRunner runner = mock(TaskRunner.class);
     when(toolbox.getTaskRunner()).thenReturn(Optional.absent());
     action.perform(task, toolbox);
     verify(runner, never()).updateStatus(any(), any());
   }
+
+  @Test
+  public void testEquals()
+  {
+    UpdateStatusAction one = new UpdateStatusAction("", TaskStatus.failure(ID, "error"));
```

Comment:
`EqualsVerifier.forClass(UpdateStatusAction.class).verify()` will verify all combinations of the underlying fields in the class and point out subtle issues in the equals / hash code logic - see other tests like `testEquals` or `testEqualsAndHashCode` for examples.